### PR TITLE
Feats/feedback loop

### PR DESCRIPTION
**Pull request by rafaljanwojcik**
### Proposed Changes:
- Working feedbacks in the UI
- feedbacks exploration page in the UI streamlit app
- included deployment change in docker-compose.yml for rest-api

### How did you test it?
locally with makefile & using the UI

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -50,4 +52,4 @@ services:
       - DEFAULT_QUESTION_AT_STARTUP
       - DEFAULT_DOCS_FROM_RETRIEVER
       - DEFAULT_NUMBER_OF_ANSWERS
-    command: "/bin/bash -c 'sleep 15 && python -m streamlit run ui/webapp.py'"
+    command: "/bin/bash -c 'pip install streamlit-aggrid plotly && sleep 15 && python -m streamlit run ui/webapp.py'"
```
Comment (quig) [inline docker-compose.yml:55]: why would we need to install a requirement in the docker compose file ?

File: ui/utils.py [hunk 2]
```diff
@@ -108,17 +108,18 @@ def send_feedback(query, answer_obj, is_correct_answer, is_correct_document, doc
...
-        raise ValueError(f"An error was returned [code {response_raw.status_code}]: {response_raw.json()}")
+        raise ValueError("An error was returned [code {}]: {}".format(response_raw.status_code, response_raw.text))
 
 
-def send_feedback_faq(*args, **kwargs):
-    # TODO
-    raise NotImplementedError
+def get_feedbacks(index_option: str):
+    url = f"{API_ENDPOINT}/{DOC_FEEDBACK}"
+    response_raw = requests.get(url, params={"index_option": index_option})
```
Comment (quig) [inline ui/utils.py:121]: I expected a POST request somehow similar to the function just above, is this get handled by the backend ?

**Review by quig — APPROVED** [review]
Thanks @rafaljanwojcik , this will be awesome for the demo  👍

File: docker-compose.yml (hunk 1 above)
Comment (rafaljanwojcik) [reply docker-compose.yml:55]: I know it's a very hacky way of doing it, but I didn't want to modify original images, and these were the only two new dependencies nessecary for the feedback exploration page in streamlit, so I added them like that 🙈

File: ui/utils.py (hunk 2 above)
Comment (rafaljanwojcik) [reply ui/utils.py:121]: Why? Retrieving feedbacks is done with a GET request in api definition. Here are the docs of the API (they're also available on your machine after starting the haystack-api docker):
https://haystack.deepset.ai/components/rest-api-definition

It's the Feedbacks endpoint

Comment (quig) [reply ui/utils.py:121]: sorry I read `send_feedbacks` 🙈 
must be quite tired...

Comment (rafaljanwojcik) [reply ui/utils.py:121]: Hahah, no worries

**Pull request merged**